src/main/java/uk/gov/hmcts/reform/managecase/client/prd/PrdFeignClient.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.managecase.client.prd;
+
+import feign.Client;
+import feign.Request;
+import feign.Response;
+import feign.RetryableException;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSocketFactory;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class PrdFeignClient extends Client.Default {
```

Comment:
The class reads to have a Prd-specific one, but it will take effect for other downstream calls.
And, retry mechanism may or may not be good fit for all such calls.
Let's discuss interactively.

Reply:
This case has been tested. Other api calls have their own config file, and never  retry has been used in them. this does not allow them to pick any retry mechanism at the moment. But a call will be good for better understanding.